lsp-completion.el

```diff
@@ -740,7 +740,7 @@ The CLEANUP-FN will be called to cleanup."
              (fboundp 'company-mode))
         (setq-local company-abort-on-unique-match nil)
         (company-mode 1)
-        (add-to-list 'company-backends 'company-capf))
+        (setq-local company-backends (append '(company-capf) company-backends)))
```

Comment:
No, I don't think you should use `append` or else it would have duplicate company backends if user already has one.

Reply:
Right, I've changed it to use `cl-adjoin` which I believe should also respect the order of the original list as well.